charts/kargo/values.yaml

```diff
@@ -171,6 +171,9 @@ api:
         # - bob@examples.com
         # groups:
         # - kargo-admin
+      ## @param api.oidc.admins.claimsv2 Subjects having any of these claims will automatically be Kargo admins.
+      # NOTE: This is a map of string to list of strings and should be used when keys contain special characters such as colons (:).
+      claimsv2: {}
```

Comment:
See https://github.com/akuity/kargo/pull/5002/files#r2328346418

We could safely ditch this new field and just use the new annotation approach with the keys/vals from the existing `api.oidc.admins.claims` setting.